### PR TITLE
docs: v3.1.0 release notes

### PR DESCRIPTION
**Pull request by brian-intel**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)
- [x] No Employer Confidential details are added

## What requirement is this design document for?
v3.1.0 release notes

#**issue_number**

### CODE REVIEW

File: docs/sitemap.xml.gz
Comment (jim-wang-intel) [inline docs/sitemap.xml.gz:1]: should not check in this binary file

**Review by jim-wang-intel — COMMENTED** [review]
other than binary file not check-in; LGTM

File: docs/sitemap.xml.gz
Comment (brian-intel) [reply docs/sitemap.xml.gz:1]: I believe these files are generated with the mkdocs tools for the github pages

**Review by antoniomtz — APPROVED** [review]
LGTM

**Review by seanohair22 — APPROVED** [review]
Just reviewed, LGTM!

**Pull request merged**